back-end/Recognition/src/utils.py

```diff
@@ -74,8 +74,8 @@ def convert_labels_to_dict(labels)->dict:
     logger.info("Converting labels to dict.")
 
     labels_dict = {
-        "labels": (label.description for label in labels),
-        "scores": (label.score for label in labels),
+        "labels": [label.description for label in labels],
+        "scores": [label.score for label in labels],
```

Comment:
You could use Generator comprehension instead of list comprehension; as here tuples are faster and more memory-efficient

Reply:
make sense , i will change it now